cli/package.json

```diff
@@ -50,6 +50,7 @@
     "@sdkgen/dart-generator": "file:../dart-generator",
     "@sdkgen/parser": "file:../parser",
     "@sdkgen/typescript-generator": "file:../typescript-generator",
+    "@sdkgen/kt-generator": "file:../kt-generator",
```

Comment:
The packages should be named `kotlin-generator` and `android-runtime` for better alignment with the other packages